x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/components/dashboard_panels.tsx

```diff
@@ -199,35 +200,37 @@ const EntityStoreDashboardPanelsComponent = () => {
         </>
       )}
 
-      {entityStore.status === 'not_installed' && !isRiskScoreAvailable && (
-        // TODO: Move modal inside EnableEntityStore component, eliminating the onEnable prop in favour of forwarding the riskScoreEnabled status
-        <EnableEntityStore
-          enablements="both"
-          onEnable={() => setModalState({ visible: true })}
-          loadingRiskEngine={riskEngineInitializing}
-        />
-      )}
+      {(entityStore.status === 'not_installed' || entityStore.status === 'stopped') &&
+        !isRiskScoreAvailable && (
+          // TODO: Move modal inside EnableEntityStore component, eliminating the onEnable prop in favour of forwarding the riskScoreEnabled status
+          <EnableEntityStore
+            enablements="both"
+            onEnable={() => setModalState({ visible: true })}
+            loadingRiskEngine={riskEngineInitializing}
+          />
+        )}
 
-      {entityStore.status === 'not_installed' && isRiskScoreAvailable && (
-        <>
-          <EuiFlexItem>
-            <EnableEntityStore
-              enablements="store"
-              onEnable={() =>
-                setModalState({
-                  visible: true,
-                })
-              }
-            />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EntityAnalyticsRiskScores riskEntity={RiskScoreEntity.user} />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EntityAnalyticsRiskScores riskEntity={RiskScoreEntity.host} />
-          </EuiFlexItem>
-        </>
-      )}
+      {(entityStore.status === 'not_installed' || entityStore.status === 'stopped') &&
```

Comment:
The diff is weird. But this is the only change I made here: `|| entityStore.status === 'stopped'`